procedures/release-duty/merge-duty/merge_new_esr_branch.rst

```diff
@@ -36,141 +25,47 @@ builds (3 weeks before the release), gives enough time to everybody.
-| ow_bug.cgi?id=1340669>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1341330 <https:          | Please add mozilla-esr52 to DXR          |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1341330>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1341491 <https:          | Set ESR_NEXT in ship-it config           |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1341491>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1342117 <https:          | Set watch_all_branches to True for ESR52 |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1342117>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1342204 <https:          | Enable TC on mozilla-esr52               |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1342204>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1342431 <https:          | Turn off TC-based Android jobs on ESR52  |
-| //bugzilla.mozilla.org/sh |                                          |
-| ow_bug.cgi?id=1342431>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1343097 <https:          | Add mozilla-esr52 and comm-esr52 to      |
-| //bugzilla.mozilla.org/sh | release-notifications heroku app         |
-| ow_bug.cgi?id=1343097>`__ |                                          |
-+---------------------------+------------------------------------------+
-| `1343366 <https:          | browser_parsable_css.js fails on ESR52   |
-| //bugzilla.mozilla.org/sh | ASAN builds due to the ESR branding      |
-| ow_bug.cgi?id=1343366>`__ | changes                                  |
-+---------------------------+------------------------------------------+
+for the `tracking bug <https://bugzilla.mozilla.org/show_bug.cgi?id=1717527>`__
+look at each bug in the tree and see if it is needed in the next ESR.  Most likely it will be.
+
+Odd problems
+------------
+
+mozilla-version needed an `update <https://github.com/mozilla-releng/mozilla-version/commit/3d9f3361505fbb485ea6103c2be6e2a8a4d41ec1>`__.
+ * remember to push mozilla-version changes to production branch, don't leave on master only
+ * remember to update `treescript <https://github.com/mozilla-releng/scriptworker-scripts/commit/d0ffb3c1c0095798c50e0f126e47280404b720ed>`__
+ * remember to merge scriptworker-script (treescript) changes to production and let the ci-change bot complete
+
+UVNEXT* tasks (update verify next) will fail until the first build is complete
```

Comment:
I don't think this is quite correct.  UVnext tasks happen in the promote phase for the build (i.e. until the first build is complete, they don't run), but rely on balrog rules being correctly set up on the esr-localtest-next channel (which they weren't initially this time around).